pkg/apis/serving/v1alpha2/configmap.go

```diff
@@ -0,0 +1,53 @@
+package v1alpha2
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/kubeflow/kfserving/pkg/constants"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetKFServiceConfigMap(client client.Client) (*v1.ConfigMap, error) {
+	configMap := &v1.ConfigMap{}
+	err := client.Get(context.TODO(), types.NamespacedName{Name: constants.KFServiceConfigMapName, Namespace: constants.KFServingNamespace}, configMap)
+	if err != nil {
+		return nil, err
+	}
+	return configMap, nil
+}
+
+func GetPredictorConfigs(configMap *v1.ConfigMap) (*PredictorsConfig, error) {
```

Comment:
Should we make a typed configmap wrapper with getters instead of these static helpers?

Reply:
done